core/src/main/java/com/graphhopper/routing/util/CarFlagEncoder.java

```diff
@@ -37,6 +37,9 @@
 
```

Comment:
Cannot comment on the faundau_destination.osm.xml file somehow ... so here it goes :)

Can we avoid another osm xml? E.g. I found destination tags in krems.osm.gz or north-bayreuth.osm.gz